cypress.json

```diff
@@ -10,6 +10,5 @@
   "supportFile": "tests/_support",
   "waitForAnimations": true,
   "videoUploadOnPasses": false,
-  "numTestsKeptInMemory": 0,
```

Comment:
What' s the advantage of having this in tests-functions and not here?

Reply:
that if you `npm run cypress:open` you actually get snapshots!

i'll spike a little on jenkins and CI and want to adjust configuration in a way that makes cypress "dev-first" (without sacrificing on the security it gives us on CI of course).

i think if it's easier to write integration tests, we'll have more of those, which should give us even more confidence in our stuff.